src/nav.config.ts

```diff
@@ -2526,10 +2526,6 @@ const fullNav: FullNav = {
           title: 'Api.Route.Patch()',
           href: '/reference/go/api/api-route-patch',
         },
-        {
-          title: 'Api.Details()',
-          href: '/reference/go/api/api-details',
-        },
```

Comment:
I'm guessing this hasn't been implemented in go? Should we raise an issue to add this method?

Reply:
It got removed in the v1 rewrite. Not sure if we still need it